plugins/README.md

```diff
@@ -0,0 +1,88 @@
+## Relayer Plugins
+
+Relayer plugins are TypeScript functions that can be invoked through the relayer HTTP API.
+
+Under the hood, the relayer will execute the plugin code in a separate process using `ts-node` and communicate with it through a Unix socket.
+
+## Setup
+
+#### 1. Writing your plugin
+
+```typescript
+import { Plugin, runPlugin } from "./lib/plugin";
+
+type Args = {
+  foo: string;
+  bar: number;
+}
+
+async function myPlugin(plugin: Plugin, args: Args) {
+    console.log(args.foo);
```

Comment:
maybe to add comment that this is for debugging purposes

Reply:
Actually I considered that part of the docs 😅 just demonstrating that your custom `foo` parameter is accessible inside the plugin